submissions/aL0NEW0LF/main.c

```diff
@@ -0,0 +1,344 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+// #include <windows.h> // YES WINDOWS :) IKR
+#include <pthread.h>
+
+// SPECIFY THE NUMBER OF ROWS IN HERE (EX FOR THE CHALLENGE TESTING DATA: 1000000000)
+#define MAX_ROWS 1000000000
+// SPECIFY THE MAXIMUM LENGTH OF city
+#define MAX_CITY_NAME_LENGTH 50
+// SPECIFY THE MAXIMUM LENGTH OF PRODUCT city
+#define MAX_PRODUCT_NAME_LENGTH 50
+// SPECIFY THE INPUT FILE PATH
+#define INPUT_FILE_PATH "input.csv"
```

Comment:
The input file name is input.txt. Please check readme

Reply:
Done.